### DISCUSSION TITLE
Extend "Keyboard shortcuts"'s feature - Ability to call Custom function

### DISCUSSION THREAD

**Discussion by gunir** [Floorp-Projects/Floorp#483] [Feature Request]
If you have heard about [keyconfig](https://kb.mozillazine.org/Keyconfig_extension), this XUL addon allow you to add/modify custom hotkey, but even more it can add hotkeys that call custom function which is very powerful.

Being able to add custom function is very beneficial because:
- It will allow us to quick test new code, because we won't need to recompile and run anymore, just write a shortcut and run custom function instead.
- Very extendable, because we can add literally anything

**Comment by Ryosuke-Asano** [comment]
Good.

**Comment by Ryosuke-Asano** [comment]
Implemented. Will be enabled on Floorp 11.5.0

**Reply by Orcastraw** [reply]
is there documentation anywhere?
